activities/GameOfLife.activity/lib/sugar-web/activity/activity.js

```diff
@@ -43,7 +43,7 @@ define(["webL10n",
         }
         bus.onNotification("activity.stop", sendStopEvent);
 
-        datastoreObject = new datastore.DatastoreObject();
+        datastoreObject = new datastore.DatastoreObject('game_of_life');
```

Comment:
Not at all a good idea. It forbid to have multiple instances of your activity.

Reply:
OK